tests/cylc.scheduler_cli/00-host-appointer.t

```diff
@@ -18,7 +18,24 @@
 # Run unit tests to test HostAppointer class for selecting hosts.
 . "$(dirname "$0")/test_header"
 #-------------------------------------------------------------------------------
-set_test_number 1
+set_test_number 3
 
 run_ok "${TEST_NAME_BASE}" python -m 'cylc.scheduler_cli'
+
+# No run hosts list
```

Comment:
This check is now in the unit tests, if you would prefer it here just delete the unit test https://github.com/cylc/cylc/pull/2693/commits/deb7d769b18d1f9bba4f3209dfadc67cd88fd240.